docs/documentation/api.md

```diff
@@ -177,7 +186,7 @@ Just as in the usage examples, the `/messages/render` key must match a
 URL definition in `zerver/openapi/zulip.yaml`, and that URL definition
 must have a `post` HTTP method defined.
 
-### Displaying example payloads/responses
+### Displaying example payloads or responses
```

Comment:
replaced the forward-slash with or as the build checker was unable to detect the navigation link with the slash